Automattic/wp-calypso
# Jetpack Search: Display in My Plans page

Partially addressees #39858; the design is not final.

<img width="1059" alt="Screen Shot 2020-03-23 at 5 19 09 PM" src="https://user-images.githubusercontent.com/4044428/77372132-9d19f700-6d2a-11ea-9e79-7d4e2a183f47.png">

#### Changes proposed in this Pull Request

* Adds support for showing Jetpack Search in the My Plans page.

#### Testing instructions

1. Spin up this branch locally.

2. Navigate to `/plans/my-plan/` and select a non-Professional Jetpack site with a Jetpack Search product subscription. Ensure that you see a card for Jetpack Search.
